google/iam/admin/v1/iam_gapic.yaml

```diff
@@ -0,0 +1,248 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.iam.admin.spi.v1
+  python:
+    package_name: google.cloud.iam.admin.v1
+  go:
+    package_name: cloud.google.com/go/iam/admin/apiv1
+  csharp:
+    package_name: Google.Iam.Admin.V1
+  ruby:
+    package_name: Google::Cloud::Iam::Admin::V1
+  php:
+    package_name: Google\Cloud\Iam\Admin\V1
+  nodejs:
+    package_name: '@google-cloud/admin'
```

Comment:
Oh, this is probably a bug on configgen to find the right name, but it's going to be `'@google-cloud/iam'`, not `admin`.